ts/components/LabelledItem/index.tsx

```diff
@@ -197,13 +191,19 @@ export const LabelledItem: React.FC<Props> = ({
               icon={props.icon}
               iconColor={iconColor}
               iconStyle={props.iconStyle}
+              accessible={false}
```

Comment:
Why we want to explicitly disable the accessibility?

Reply:
This icon being a `ButtonDefaultOpacity` are spoken aloud as button and an eventually `accessibilityLabelIcon` passed as prop is not read. A better handling of this component is out of the scope of this issue. Anyway is well documented here: https://pagopa.atlassian.net/browse/IABT-1415

Reply:
For this reason we have decided for now to disable its accessibility.